integration/build/build_test.go

```diff
@@ -307,9 +307,8 @@ COPY bar /`
 // docker/for-linux#135
 // #35641
 func TestBuildMultiStageLayerLeak(t *testing.T) {
-	fmt.Println(testEnv.DaemonAPIVersion())
-	skip.IfCondition(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.38"),
-		"Don't run on API lower than 1.38 as it has been fixed starting from that version")
+	skip.IfCondition(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.37"),
+		"Don't run on API lower than 1.37 as it has been fixed starting from that version")
```

Comment:
what about this one!

Reply:
so... in the previous PR I thought it was fixed only starting 1.38.. but @thaJeztah told me it's been backported so already fixed for 1.37 😅 😹

Reply:
Sorry, I mean't that you shortened the comment above to say "Broken in earlier versions", and this one could be shortened as well I think.

Reply:
ah right, fixed stuck_out_tongue_closed_eyes